src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/PrintableFormServiceChannelInTestsIT.java

```diff
@@ -0,0 +1,185 @@
+package fi.otavanopisto.kuntaapi.test.server.integration.ptv;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.jayway.restassured.http.ContentType;
+
+import fi.metatavu.kuntaapi.server.rest.model.PrintableFormServiceChannel;
+import fi.metatavu.kuntaapi.server.rest.model.ServiceChannelAttachment;
+import fi.metatavu.ptv.client.model.V7VmOpenApiAddressDeliveryIn;
+import fi.metatavu.ptv.client.model.V7VmOpenApiPrintableFormChannel;
+import fi.metatavu.ptv.client.model.V7VmOpenApiPrintableFormChannelInBase;
+import fi.metatavu.ptv.client.model.VmOpenApiAddressPostOfficeBoxIn;
+import fi.metatavu.ptv.client.model.VmOpenApiAddressStreetIn;
+import fi.metatavu.ptv.client.model.VmOpenApiAttachment;
+import fi.metatavu.ptv.client.model.VmOpenApiLanguageItem;
+import fi.otavanopisto.kuntaapi.server.integrations.ptv.translation.PtvAddressSubtype;
+import fi.otavanopisto.kuntaapi.server.persistence.model.clients.AccessType;
+import fi.otavanopisto.kuntaapi.server.persistence.model.clients.ClientOrganizationPermission;
+import fi.otavanopisto.kuntaapi.test.AbstractPtvMocker;
+
+public class PrintableFormServiceChannelInTestsIT extends AbstractPtvInTest {
+
+  private static final String SERVICE_CHANNEL_FIND_PATH = "/printableFormServiceChannels/{kuntaApiChannelId}";
+  /**
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)